pipeline/ephys.py

```diff
@@ -629,6 +629,21 @@ class WaveformMetric(dj.Part):
         velocity_below=null: float   
         """
 
+
+@schema
+class UnitPassingCriteria(dj.Computed):
+    """ This table computes whether a unit_key passes a selection criteria """
+    definition = """
+    -> Unit
+    ---
+    criteria_passed : bool      #boolean based on criteria
+    """
+
```

Comment:
Will need to modify the `key_source` here, the function `check_unit_criteria` requires many tables downstream from `Unit` to be populated, e.g. UnitStat, ClusterMetric, etc.
Please add a `key_source` that ensure those other required tables are populated before this one